modules/ding_provider/connie/connie.module

```diff
@@ -141,8 +141,18 @@ function connie_availability_holdings($provider_ids) {
   foreach ($res as $id => &$availability) {
     if (((int) ($id / 100) % 2) == 0) {
       $availability['holdings'] = array(
-        'Andeby bibliotek',
-        'Gåserød bibliotek',
+        'Andeby bibliotek' => [
+          'reservable' => TRUE,
+          'total_count' => 10,
+          'available' => TRUE,
+          'ordered_count' => 2,
+        ],
+        'Gåserød bibliotek' => [
+          'reservable' => TRUE,
+          'total_count' => 10,
+          'available' => TRUE,
+          'ordered_count' => 2,
```

Comment:
Having different values for `reservable`, `total_count` etc. for the different branch holdings would be useful.